lib/bounding-boxes.ts

```diff
@@ -26,6 +26,15 @@ export const combineBoundingBoxes = (
   );
 };
 
+export const getBoundingBoxCenter = (boundingBox: BoundingBox): {latitude: number, longitude: number} => {
+  const [minLng, minLat, maxLng, maxLat] = boundingBox;
+
+  return {
+    latitude: (maxLat + minLat) / 2,
+    longitude: (maxLng + minLng) / 2
```

Comment:
On very rare occasions (Malaysia) this will be in the water but I swear it really doesn't bother me and it's really not obvious.